src/test/java/org/broadinstitute/hellbender/MainTest.java

```diff
@@ -1,13 +1,65 @@
 package org.broadinstitute.hellbender;
 
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.TestProgramGroup;
 import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.junit.Assert;
 import org.testng.annotations.Test;
 
-public final class MainTest extends CommandLineProgramTest{
+import java.util.Collections;
+import java.util.List;
+
+public final class MainTest extends CommandLineProgramTest {
 
     @Test(expectedExceptions = UserException.class)
     public void testCommandNotFoundThrows(){
         this.runCommandLine(new String[]{"Brain"});
     }
+
+
+    /** Main class for testing a single CLP */
+    private static final class SingleClpMain extends Main {
+
+        private Class<? extends CommandLineProgram> clazz;
+
+        private SingleClpMain(final Class<? extends CommandLineProgram> clazz) {
+            this.clazz = clazz;
+        }
+
+        @Override
+        protected List<String> getPackageList() {
+            return Collections.emptyList();
+        }
+
```

Comment:
I think lines 25-35 are unnecessary if you make getClassList just return OmitFromCommanLineCLP.class.

Reply:
Also, please change OmitFromCommanLineCLP -> OmitFromCommandLineCLP, it has a missing "d" everywhere it appears.

Reply:
I implemented the `SingleClpMain` as a helper class for increase the coverage of `Main` related with CLPs.  This allow to test in a more granular way different parts of the code paths handling CLPs, by implementing a new CLP with the required behaviour, and construct a `SingleClpMain` that is ensured to have only one CLP. For instance, this will allow to test the complete usage message. I am preparing another PR to increase the coverage of `Main` and I think that this will be helpful. Let me know what do you think about it.

Regarding the typo, it is fixed in the next commit.

Reply:
If its going to be used in another PR, lets include/review it there rather than in this PR.

Reply:
Ok, done.